### PR TITLE
kernel: bump 6.1 to 6.1.57

### PR DESCRIPTION
**Pull request by graysky2**
Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.57
```
Manually rebased:
	generic/pending-6.1/702-net-ethernet-mtk_eth_soc-enable-threaded-NAPI.patch

Removed upstreamed:
	qualcommax/patches-6.1/0134-PCI-qcom-Fixing-broken-pcie-enumeration-for-2_3_3-co.patch[1]
```
All other patches automatically rebased.

1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.1.57&id=2dfb5f324d799f4545e17631415aba6d302a8e2b

Build system: x86/64
Build-tested: x86/64/AMD Cezanne
Run-tested: x86/64/AMD Cezanne

### CODE REVIEW

File: target/linux/generic/pending-6.1/702-net-ethernet-mtk_eth_soc-enable-threaded-NAPI.patch [hunk 1]
```diff
@@ -10,26 +10,6 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
...
-@@ -3168,8 +3168,8 @@ static irqreturn_t mtk_handle_irq_tx(int
- 
- 	eth->tx_events++;
- 	if (likely(napi_schedule_prep(&eth->tx_napi))) {
--		__napi_schedule(&eth->tx_napi);
- 		mtk_tx_irq_disable(eth, MTK_TX_DONE_INT);
-+		__napi_schedule(&eth->tx_napi);
- 	}
- 
- 	return IRQ_HANDLED;
```
Comment (Ansuel) [inline target/linux/generic/pending-6.1/702-net-ethernet-mtk_eth_soc-enable-threaded-NAPI.patch:32]: LOL... no idea we had this change... I accidentally pushed the same change while reworking napi_schedule and noticing the problem in how it was handled.

Comment (graysky2) [reply target/linux/generic/pending-6.1/702-net-ethernet-mtk_eth_soc-enable-threaded-NAPI.patch:32]: How shall we proceed?  Is this PR good as-is or do you suggest some change?

Comment (Ansuel) [reply target/linux/generic/pending-6.1/702-net-ethernet-mtk_eth_soc-enable-threaded-NAPI.patch:32]: Ok as is. It just got backported. Was just me surprised noticing this and wanted to put some text about it ahhahah

Comment (graysky2) [reply target/linux/generic/pending-6.1/702-net-ethernet-mtk_eth_soc-enable-threaded-NAPI.patch:32]: Sounds good, thanks!

File: target/linux/qualcommax/config-6.1 [hunk 2]
```diff
@@ -28,6 +28,7 @@ CONFIG_ARM64_ERRATUM_2054223=y
 CONFIG_ARM64_ERRATUM_2067961=y
 CONFIG_ARM64_ERRATUM_2077057=y
 CONFIG_ARM64_ERRATUM_2658417=y
+CONFIG_ARM64_ERRATUM_2966298=y
```
Comment (robimarko) [inline target/linux/qualcommax/config-6.1:31]: Qualcommax is using Cortex-A53 cores, so it shouldn't be affected by this at all

Comment (robimarko) [reply target/linux/qualcommax/config-6.1:31]: Same goes for all targets in OpenWrt, none of them uses A520 core, it probably makes sense to enable it for armsr only though that is targeting ARMv8

Comment (graysky2) [reply target/linux/qualcommax/config-6.1:31]: At least two checks failed which prompted me to look into the config changes.  I cannot remember which two they were though.  I think one as `armsr/armv8`... I will remove the modifications and see which ones fail.

Comment (robimarko) [reply target/linux/qualcommax/config-6.1]: They are just using the default value, for OpenWrt it should just be disabled in the generic config.

Comment (Ansuel) [reply target/linux/qualcommax/config-6.1]: (aka setting this as # ... is not defined in the generic configuration)

Comment (robimarko) [reply target/linux/qualcommax/config-6.1]: Yes, and then you can drop all of the target config changes as we dont have that core in any OpenWrt target.

Comment (graysky2) [reply target/linux/qualcommax/config-6.1]: Done.

**Comment by robimarko** [comment]
LGTM,
`Reviewed-by: Robert Marko <robimarko@gmail.com>`

**Pull request merged**